polars/polars-core/src/serde/series.rs

```diff
@@ -154,7 +151,9 @@ impl<'de> Deserialize<'de> for Series {
                     }
                     DataType::UInt32 => {
                         let values: Vec<Option<u32>> = map.next_value()?;
```

Comment:
Not related to this PR, but this may be a nice follow up. 

Currently we deserialize first into `Vec<Option<T>>` which then is copied into

```
values: Buffer<T>,
validity: Bitmap
```

We could do much better by serializing just a binary `Vec<u8>` that is serialized/deserialized with arrow IPC. This is arrow on disk (the buffers are concatenated and there is a footer which tells the reader where the offsets are).

Which means that upon deserializing we can memory map or directly copy to the proper buffers via the Ipc readers.